### ISSUE TITLE
lint doesn't recognize "language: minimal"

### ISSUE THREAD

**Issue by PiotrSikora** [travis-ci/travis.rb#504]
Calling `travis lint` results in a warning message:

    Warnings for .travis.yml:
    [x] in language section: illegal value minimal, defaulting to ruby

but the rumor says that `language: minimal` is a valid setting.

**Comment by kennytm** [comment]
There is really no `language: minimal`. Perhaps you mean `language: generic`?

**Comment by PiotrSikora** [comment]
Looking at [the source code](https://github.com/travis-ci/travis-yml/blob/master/spec/travis/yaml/spec/def/lang/shell_spec.rb), both: `minimal` and `generic` are aliases of `shell`, so both should be recognized by the `travis lint` tool.

**Comment by abitrolly** [comment]
This is fixed for 1.8.9
```
✗ travis version
1.8.9
✗ travis lint          
Hooray, .travis.yml looks valid :)
✗ cat .travis.yml
language: minimal
os:
  - linux
  - osx
install: echo "Default install step"
script: echo "Default build step"
```
But it still doesn't understand `jobs` #571 and probably other relatively new features.

**Comment by BanzaiMan** [comment]
The `lint` command has gone through substantial changes recently. If you still see this problem, please open a new issue.
